GNUmakefile

```diff
@@ -141,4 +134,4 @@ validate-examples:
 
 pr-check: generate build test lint tflint website-lint
 
-.PHONY: build build-docker test test-docker testacc vet fmt fmtcheck errcheck scaffold-website test-compile website website-test validate-examples
+.PHONY: build test testacc vet fmt fmtcheck errcheck scaffold-website test-compile website website-test validate-examples
```

Comment:
the `pr-check` target above is missing from `.PHONY` worth adding while we're here?